playbooks/luna.yml

```
@@ -21,6 +21,7 @@
       - "--enable-foreman-cli-remote-execution"
       - "--enable-foreman-cli-tasks"
       - "--enable-foreman-cli-templates"
+      - "--enable-ostree"
```

Does this work? AFAIK we don't have an ostree class. Don't you mean:
```suggestion
      - "--katello-ostree true"
```